hack/BUILD.bazel

```diff
@@ -16,6 +16,26 @@ CONTROLLER_GEN = "@io_k8s_sigs_controller_tools//cmd/controller-gen"
 
 # Bazel file generation rules
 
+sh_binary(
+    name = "update-deps",
+    srcs = ["update-deps.sh"],
+    args = [
+        "$(location %s)" % GO,
+        "$(location %s)" % GAZELLE,
+        "$(location %s)" % KAZEL,
+        "$(location :update-bazel)",
+        "$(location :update-deps-licenses)",
+    ],
+    data = [
+        GAZELLE,
+        GO,
+        KAZEL,
+        ":update-bazel",
+        ":update-deps-licenses",
+    ],
+    tags = ["manual"],
+)
```

Comment:
This is re-added from [an older commit](https://github.com/cert-manager/cert-manager/blob/3c33140c110631c69214efb3206e5b12b60112ef/hack/BUILD.bazel) but marked manual so it won't be automatically run anywhere.